foundry/pixi/containers/canvasLayers/placeablesLayer.d.ts

```diff
@@ -49,7 +49,7 @@ declare abstract class PlaceablesLayer<P extends PlaceableObject = PlaceableObje
   /**
    * A Quadtree which partitions and organizes Walls into quadrants for efficient target identification.
    */
-  quadtree: Quadtree | null;
+  quadtree: Quadtree<PlaceableObject> | null;
```

Comment:
Shouldn't this maybe be `Quadtree<P>`? I'm no 100%, but it seems to be the case for WallsLayer.

Reply:
I was going off of the return type of PlaceablesLayer.placeableClass's return type which is just PlaceableObject.

Reply:
But looking at the class more, I think you are right in that this should be P, I will get it updated.